manager/src/main/java/com/redhat/service/smartevents/manager/api/internal/ShardBridgesSyncAPI.java

```diff
@@ -82,15 +83,13 @@ public class ShardBridgesSyncAPI {
     @Operation(hidden = true, summary = "Update a Processor.", description = "Update a Processor.")
     @PUT
     @Path("processors")
-    public Response updateProcessorStatus(ProcessorDTO processorDTO) {
+    public Response updateProcessorStatus(UpdateManagedResourceStatusDTO updateDTO) {
         String shardId = identityResolver.resolve(jwt);
         failIfNotAuthorized(shardId);
-        LOGGER.info("Processing update from shard for Processor with id '{}' for bridge '{}' for customer '{}' with status '{}'",
-                processorDTO.getId(),
-                processorDTO.getBridgeId(),
-                processorDTO.getCustomerId(),
-                processorDTO.getStatus());
-        processorService.updateProcessorStatus(processorDTO);
+        LOGGER.info("Processing update from shard for Processor with id '{}' with status '{}'",
```

Comment:
Same here.
Btw customerID is also in the new bean so we can preserve it in the log